changelogs/unreleased/gh-10766-synchro-crash-on-exit.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/replication
+
+* Fixed a crash which could happen when an instance was being shut down while
+  having an unfinished synchronous transaction (gh-10766).
```

Comment:
```suggestion
* Fixed a crash which could happen while an instance was shut down
  having an unfinished synchronous transaction (gh-10766).
```

Reply:
I just suggested more lightweight phrasing: the first "while" implies that we're talking about the shutdown *process* (so we can get rid of "was being shut down" in favor of "was shut down") + the second "while" is not much needed here.